cli/src/cli.rs

```diff
@@ -1987,6 +1987,17 @@ pub fn app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> App<'ab, '
                         .takes_value(true)
                         .required(true)
                         .help("The transaction signature to confirm"),
+                )
+                .after_help(// Formatted specifically for the manually-indented heredoc string
+                   "Note: This will show more detailed information for finalized transactions with verbose mode (-v/--verbose).\
+                  \n\
+                  \nAccount's mode legend in the details:\
+                  \n  |srwx|\
+                  \n    s: this account correctly signed\
+                  \n    r: this account is marked to be read (always true)\
+                  \n    w: this account is marked to be written\
+                  \n    x: this account is marked to be executed via program_id (CPI's program_id can't be marked, though)\
```

Comment:
```suggestion
                  \n    s: account signed the transaction \
                  \n    r: account is readable \
                  \n    w: account is writable \
                  \n    x: program account (inner instructions excluded)
```

Reply:
Just saw this suggestion. I like `program account (inner instructions excluded)` better than what I suggested